rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+DCP SHOULD NOT duplicate the ZenHub DCP board in other documents or spreadsheets. The board and its reports represent the definitive list of DCP prioritized tasks (Product Backlog) and their state.
+
+### ! Scrum → ZenHub Secret Decoder Ring
+
+* Backlog items → Issues with User Stories
+* Spikes → Issues with a "Spike" GitHub label
+* Epics → Epics
+* Sprint → Milestone
+* Product backlog → Issues without a Milestone in the *Product Backlog* pipeline
+* Sprint Backlog → Issues with a Milestone in the *Sprint Backlog* pipeline
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the ordered **!** **Scrum** Product Backlog for DCP. 
+
+The top-level *dcp* repo MUST contain **all** significant features that are important to the DCP community as a whole and require visibility and prioritization. This includes  issues or epics describing user features or product-wide technical requirements (e.g. “components should have logs...“). 
+
+All component repos that will assign items to a DCP Milestone MUST be added to the ZenHub *DCP Backlogs* workspace.
+
+DCP Product Backlog items  are modeled as:
+- [User Stories](http://agiledictionary.com/277/user-story/) with acceptance criteria
+- [Epics](http://agiledictionary.com/309/epic/) contain a related set of smaller User Stories required to complete an objective.
+- [Spikes](http://agiledictionary.com/209/spike/) adopted from eXtreme Programming to answer design questions.
+  - [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+#### User Stories
+
+The [*researcher with a keyboard*](https://docs.google.com/drawings/d/1Li9o_uVkEZDZHbkQv86I49FG8I0zGSgHYkKabp7piEs/edit) and [*researcher with a pipette*](https://docs.google.com/drawings/d/1o6I5BLtJuRJ1vUdb7dHCkFbo0XiaxKDBiVAogqnUowU/edit) journeys are the initial source of user stories that  are modeled as epics in the *dcp* repo,  reflecting the narrative flow of the product. Further refinement by the Product Owners is required to both develop and  extend the high-level set of user activities. 
+
+Since journeys may vary per persona, personas MUST be identified for an epic by assigning a documented GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
+
+#### Technical and Operational Requirements
+Product-wide technical or operational requirements are defined by **Technical Architecture** and modeled as epics in the *dcp* repo.
+
+#### Owners
+
+A DCP owner MUST be assigned to each issue or epic in the *Product Backlog* in the *dcp* repo to shepherd and coordinate progress. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to epics in the *dcp* repo.
```

Comment:
What do you think of changing this to require an owner for an epic to transition to the "Sprint Backlog" pipeline, but not for every epic in the backlog? If the backlog is long then we could have epics sufficiently far away, and vague enough to not have an owner yet. They may require a spike for clarification first. "Support for big 'R' data release" might be a good example of this.

Reply:
We have _Icebox_ for future, ambiguous issues that do not require a shepherd yet - but I would suggest that it's critical to have an assigned shepherd for all Product Backlog items to ensure that they're being refined and progressing towards the Sprint Backlog.